isthmus/src/main/java/io/substrait/isthmus/expression/LiteralConstructorConverter.java

```diff
@@ -25,13 +25,13 @@ public Optional<Expression> convert(
           ExpressionCreator.list(
               false,
               call.operands.stream()
-                  .map(t -> ((Expression.Literal) t.accept(new RexExpressionConverter())))
+                  .map(t -> ((Expression.Literal) topLevelConverter.apply(t)))
```

Comment:
Avoiding the use of `new RexExpressionConverter()`
This constructor doesn't setup all the callConverters.

Reply:
On second thought the `new RexExpressionConverter()` might have been deliberate to avoid recursively triggering function conversion.

Please let me know...